pyiron_atomistics/lammps/interactive.py

```diff
@@ -51,11 +51,14 @@ def structure(self, structure):
         self._prism = UnfoldingPrism(structure.cell)
         GenericInteractive.structure.fset(self, structure)
 
-    def get_structure(self, iteration_step=-1, wrap_atoms=True):
-        return super(GenericInteractive, self).get_structure(
+    def _get_structure(self, iteration_step=-1, wrap_atoms=True):
```

Comment:
I checked out the branch and it looks to me like `super()._get_structure` and `_number_of_structures` trace back to the methods defined on `AtomisticGenericJob`; is this correct? If so, why does the `super` call need an explicit parent path, and why are these methods needed at all since they're just naked supers calls?

Reply:
Back then, I just renamed it blindly.  But I think you're right both inheritance paths go to `AtomisticGenericJob`, so the super calls should be unnecessary.